From b4733921b295fd9e5067b90df9f778eebabfd948 Mon Sep 17 00:00:00 2001
From: KoysX <2685697179b@gmail.com>
Date: Tue, 4 Jul 2023 23:43:00 +0800
Subject: [PATCH] Allow-disabling-of-fingerprint-cleanups-needed-on-so

---
 .../biometrics/sensors/fingerprint/hidl/Fingerprint21.java      | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/services/core/java/com/android/server/biometrics/sensors/fingerprint/hidl/Fingerprint21.java b/services/core/java/com/android/server/biometrics/sensors/fingerprint/hidl/Fingerprint21.java
index f93281e0..f93b3500 100644
--- a/services/core/java/com/android/server/biometrics/sensors/fingerprint/hidl/Fingerprint21.java
+++ b/services/core/java/com/android/server/biometrics/sensors/fingerprint/hidl/Fingerprint21.java
@@ -734,6 +734,8 @@ public class Fingerprint21 implements IHwBinder.DeathRecipient, ServiceProvider
 
     private void scheduleInternalCleanup(int userId,
             @Nullable ClientMonitorCallback callback) {
+	final boolean mNocleanup = android.os.SystemProperties.getBoolean("persist.sys.phh.fingerprint.nocleanup", false);
+	if(mNocleanup) return;
         mHandler.post(() -> {
             scheduleUpdateActiveUserWithoutHandler(userId);
 
-- 
2.20.1

