From dbb17aedeafd2c5094e776c5521e00a466b63f4a Mon Sep 17 00:00:00 2001
From: KoysX <2685697179b@gmail.com>
Date: Sat, 8 Jul 2023 15:06:03 +0800
Subject: [PATCH] Add-Mediatek-power-hints-on-touch-moudle

---
 services/core/Android.bp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/services/core/Android.bp b/services/core/Android.bp
index 8ad1fec9..1d9f3c27 100644
--- a/services/core/Android.bp
+++ b/services/core/Android.bp
@@ -180,6 +180,7 @@ java_library_static {
         "kotlinx_coroutines",
         "faceunlock_framework",
         // HIDL
+        "vendor.mediatek.hardware.mtkpower-V1.1-java",
         "vendor.samsung.hardware.sysinput-V1.2-java",
         "vendor.goodix.hardware.biometrics.fingerprint-V2.1-java",
         "vendor.samsung.hardware.biometrics.fingerprint-V3.0-java",
-- 
2.20.1

