From f1590578e4ade71124a3e14f3be81206ee9d8260 Mon Sep 17 00:00:00 2001
From: Kuaipz <2685697179@qq.com>
Date: Sat, 27 May 2023 20:30:32 +0800
Subject: [PATCH] Every-voldmanaged-storage-is-adoptable

---
 main.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/main.cpp b/main.cpp
index 5b1bdf7..e33570b 100644
--- a/main.cpp
+++ b/main.cpp
@@ -273,7 +273,7 @@ static int process_config(VolumeManager* vm, VoldConfigs* configs) {
             int partnum = entry.partnum;
             int flags = 0;
 
-            if (entry.is_encryptable()) {
+            if (entry.is_encryptable() || true) {
                 flags |= android::vold::Disk::Flags::kAdoptable;
                 configs->has_adoptable = true;
             }
-- 
2.40.1

