From ad2c0edd6df75c5599a28424c5ab0ed7833431a5 Mon Sep 17 00:00:00 2001
From: Kuaipz <2685697179@qq.com>
Date: Sat, 27 May 2023 20:35:18 +0800
Subject: [PATCH] init-Do-not-start-console-service-when-debuggable

---
 rootdir/init.rc | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/rootdir/init.rc b/rootdir/init.rc
index c26f83e..1282842 100644
--- a/rootdir/init.rc
+++ b/rootdir/init.rc
@@ -1267,9 +1267,6 @@ on property:ro.debuggable=1
     # Give reads to anyone for the accessibility trace folder on debug builds.
     chmod 0775 /data/misc/a11ytrace
 
-on init && property:ro.debuggable=1 && property:ro.console.enable=1
-    start console
-
 on userspace-reboot-requested
   # TODO(b/135984674): reset all necessary properties here.
   setprop sys.boot_completed ""
-- 
2.40.1

