From 6908f9253afdd5eeb0242bb8d1653de9c46dd752 Mon Sep 17 00:00:00 2001
From: KoysX <2685697179b@gmail.com>
Date: Sat, 8 Jul 2023 14:29:57 +0800
Subject: [PATCH] Revert-Set-system-xbin-permissions-to-750

---
 libcutils/fs_config.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libcutils/fs_config.cpp b/libcutils/fs_config.cpp
index 342f9ae..d3c2c1f 100644
--- a/libcutils/fs_config.cpp
+++ b/libcutils/fs_config.cpp
@@ -90,7 +90,7 @@ static const struct fs_path_config android_dirs[] = {
     { 00755, AID_ROOT,         AID_SHELL,        0, "system/etc" },
     { 00755, AID_ROOT,         AID_ROOT,         0, "system/etc/ppp" },
     { 00755, AID_ROOT,         AID_SHELL,        0, "system/vendor" },
-    { 00750, AID_ROOT,         AID_SHELL,        0, "system/xbin" },
+    { 00751, AID_ROOT,         AID_SHELL,        0, "system/xbin" },
     { 00751, AID_ROOT,         AID_SHELL,        0, "system/apex/*/bin" },
     { 00751, AID_ROOT,         AID_SHELL,        0, "system_ext/bin" },
     { 00751, AID_ROOT,         AID_SHELL,        0, "system_ext/apex/*/bin" },
-- 
2.20.1

