From 23ef064acc8b250b79dc74b62a77c3dd59537179 Mon Sep 17 00:00:00 2001
From: KoysX <2685697179b@gmail.com>
Date: Fri, 7 Jul 2023 09:44:09 +0800
Subject: [PATCH] fs_mgr-Keep-allowing-encryptable-fstab-flag

---
 fs_mgr/fs_mgr_fstab.cpp | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/fs_mgr/fs_mgr_fstab.cpp b/fs_mgr/fs_mgr_fstab.cpp
index fc200b2..f7436a8 100644
--- a/fs_mgr/fs_mgr_fstab.cpp
+++ b/fs_mgr/fs_mgr_fstab.cpp
@@ -315,12 +315,6 @@ bool ParseFsMgrFlags(const std::string& flags, FstabEntry* entry) {
     // some recovery fstabs still contain the FDE options since they didn't do
     // anything in recovery mode anyway (except possibly to cause the
     // reservation of a crypto footer) and thus never got removed.
-    if (entry->fs_mgr_flags.crypt && !entry->fs_mgr_flags.vold_managed &&
-        access("/system/bin/recovery", F_OK) != 0) {
-        LERROR << "FDE is no longer supported; 'encryptable' can only be used for adoptable "
-                  "storage";
-        return false;
-    }
     return true;
 }
 
-- 
2.20.1

