From 009c53930cd0e7b04fec1f97beee0fb96272f0ed Mon Sep 17 00:00:00 2001
From: Kuaipz <2685697179@qq.com>
Date: Sat, 27 May 2023 20:26:04 +0800
Subject: [PATCH] remove-su-from-no-su-builds

---
 target/product/base_system.mk | 1 -
 1 file changed, 1 deletion(-)

diff --git a/target/product/base_system.mk b/target/product/base_system.mk
index b0870c3..294bf58 100644
--- a/target/product/base_system.mk
+++ b/target/product/base_system.mk
@@ -390,7 +390,6 @@ PRODUCT_PACKAGES_DEBUG := \
     ss \
     start_with_lockagent \
     strace \
-    su \
     sanitizer-status \
     tracepath \
     tracepath6 \
-- 
2.40.1

