From e29318ae4f08703629b529fa653f150cc8acd7b0 Mon Sep 17 00:00:00 2001
From: Kuaipz <2685697179@qq.com>
Date: Sat, 27 May 2023 20:14:27 +0800
Subject: [PATCH] Address-Huawei-offline-charge-sepolicy

---
 sepolicy/charger.te | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/sepolicy/charger.te b/sepolicy/charger.te
index cbcb255..5efdd95 100644
--- a/sepolicy/charger.te
+++ b/sepolicy/charger.te
@@ -13,3 +13,9 @@ allow charger sysfs_power:lnk_file read;
 
 # The system charger can write  powerctl properties
 set_prop(charger, powerctl_prop)
+
+# Allow charger to write to sysfs_backlight_attr
+allow charger sysfs_backlight_attr:file w_file_perms;
+
+# Allow charger to write to sysfs_led_attr
+allow charger sysfs_led_attr:file w_file_perms;
-- 
2.40.1

